win32ss/printing/base/winspool/printers.c

```diff
@@ -580,13 +670,15 @@ BOOL WINAPI
 GetPrinterA(HANDLE hPrinter, DWORD Level, LPBYTE pPrinter, DWORD cbBuf, LPDWORD pcbNeeded)
 {
     TRACE("GetPrinterA(%p, %lu, %p, %lu, %p)\n", hPrinter, Level, pPrinter, cbBuf, pcbNeeded);
+    if(pcbNeeded) *pcbNeeded = 0;
```

Comment:
Hmm it seems strange that this shows up here given that it's already in master?

Reply:
I agree. I tried to do a "Rsync", but it did not appear to work correctly. Suggestions welcomed.